packages/shared/src/components/Icon.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+
+export type Size = 'small' | 'medium' | 'large';
+
+const IconSize = {
+  small: 5,
+  medium: 6,
+  large: 8,
+};
```

Comment:
When using tailwindcss with dynamic classes, you must not concatenate the value manually. You must have the whole value of the expected class as the former will get ignored by tree-shaking.
```suggestion
const IconSize = {
  small: 'w-5 h-5',
  medium: 'w-6 h-6',
  large: 'w-8 h-8',
};
```

Reply:
Got it

Reply:
Done